Failing to install npm dependencies for `ui/changes`

Currently the CI is borking due to the following error:
```
+ npm install --no-progress
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! code 126
npm ERR! path /bugbug/ui/changes/node_modules/utf-8-validate
npm ERR! command failed
npm ERR! command sh -c node-gyp-build
npm ERR! sh: 1: node-gyp-build: Text file busy
npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-06-12T04_01_02_223Z-debug-0.log
```

--- a/.taskcluster.yml
+++ b/.taskcluster.yml
@@ -213,7 +213,7 @@ tasks:
         workerType: batch
         payload:
           maxRunTime: 3600
-          image: node
+          image: node:lts
           command:
             - "/bin/sh"
             - "-lcxe"

